wraith/logged_out.yaml

```diff
@@ -0,0 +1,202 @@
+  posts_search:
+    path: /posts/search
+    selector: '[id=content]'
+
+  replies_search:
+    path: /replies/search
+    selector: '[id=content]'
+
+  report_daily:
+    path: /reports/daily
+    selector: '[id=content]'
+
+  tags:
+    path: /tags
+    selector: '[id=content]'
+  tags_settings:
+    path: /tags?view=Setting
+    selector: '[id=content]'
+  tag_show:
+    path: /tags/2
+    selector: '[id=content]'
+
+  template_show:
+    path: /templates/11
+    selector: '[id=content]'
+  template_list:
+    path: /templates/11?view=list
+    selector: '[id=content]'
+  templates_search:
+    path: /templates/search
+    selector: '[id=content]'
+
+  users:
+    path: /users
+    selector: '[id=content]'
+  user_show:
+    path: /users/3
+    selector: '[id=content]'
+  user_boards:
+    path: /users/3/boards
+    selector: '[id=content]'
+  users_search:
+    path: /users/search
+    selector: '[id=content]'
+
+
+# (optional) JavaScript file to execute before taking screenshot of every path. Default: nil
+before_capture: 'javascript/accept_tos.js'
+
+# (required) Screen widths (and optional height) to resize the browser to before taking the screenshot.
+screen_widths:
+  - 750x1334
+  - 1024x768
+  - 1920x1080
+
+# (optional) Resize to each screen width (efficient), or reload at each screen width (costly). Default: 'reload'
+resize_or_reload: 'resize'
+
+# (required for history mode, otherwise optional) The directory that your base screenshots will be stored in.
+history_dir: 'then/out'
+
+# (required) The directory that your latest screenshots will be stored in
+directory: 'now/out'
+
+# (required) Amount of fuzz ImageMagick will use when comparing images. A higher fuzz makes the comparison less strict.
+fuzz: '20%'
+
+# (optional) The maximum acceptable level of difference (in %) between two images before Wraith reports a failure. Default: 0
+threshold: 5
+
+# (optional) Specify the template (and generated thumbnail sizes) for the gallery output.
+gallery:
+  template: 'slideshow_template' # Examples: 'basic_template' (default), 'slideshow_template'
+  thumb_width:  200
+  thumb_height: 200
+
+# (optional) Choose which results are displayed in the gallery, and in what order. Default: alphanumeric
+# Options:
+#   alphanumeric - all paths (with or without a difference) are shown, sorted by path
+#   diffs_first - all paths (with or without a difference) are shown, sorted by difference size (largest first)
+#   diffs_only - only paths with a difference are shown, sorted by difference size (largest first)
+# Note: different screen widths are always grouped together.
+mode: diffs_first
+
+# (optional) Choose to run Wraith in verbose mode, for easier debugging. Default: false
+#verbose: true
+
+# (optional) Color to highlight the image diff. Default: 'blue'
+highlight_color: red
+
+# (optional) Parameters to pass to Phantom/Casper command line. Default: '--ignore-ssl-errors=true --ssl-protocol=tlsv1'
+phantomjs_options: ''
```

Comment:
does this need the ssl thing? does core? they should probably match. or we should have base options elsewhere they both import.